docs/core-protocol/public-lock/renewals.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Renewals
+description: One of the unique characteristics of the Unlock Protocol is that it can create NFT that have an expiration date. Once expired these NFT can be renewed, or extended, providing the ability to do subscriptions!
```

Comment:
```suggestion
description: One of the unique characteristics of the Unlock Protocol is that it can create NFTs that have expiration dates. Once expired, these NFTs can be renewed, or extended, providing the ability to do subscriptions!
```